files/tests/pre-merge/amazonlinux2-cabal-_l.containerfile

```diff
@@ -0,0 +1,35 @@
+FROM amazonlinux:2
+
+ARG CNODE_HOME=/opt/cardano/cnode
+
+ENV \
+  LANG=C.UTF-8 \
+  USER=root \
+  PATH=$CNODE_HOME/scripts:/root/.cabal/bin:/root/.ghcup/bin:$PATH
+
+WORKDIR /
+
+RUN amazon-linux-extras install epel -y && yum -y install git-2.32.0-1.amzn2.0.1 && yum clean all &&\
+    mkdir -pv /root/.cabal/bin /root/.ghcup/bin &&\
+    curl -sS -o prereqs.sh https://raw.githubusercontent.com/cardano-community/guild-operators/master/scripts/cnode-helper-scripts/prereqs.sh &&\
```

Comment:
I am not sure if I fully understand the use of master branch (I have not tested it on my end yet - just reading, so excuse me if I misunderstood) - we'd want to use the branch from the commit before the merge right (also, the merge would likely be targetting alpha branch - as no PR done directly against master will be sustained, they get overwritten by alpha)?

If above query is valid, the same would prolly apply to every file in this folder

Reply:
Commented just a bit ago I forgot to implement using the PR branch, vs. master. Will fix this up